chef/cookbooks/crowbar/templates/default/crowbar-upgrade-os.sh.erb

```diff
@@ -138,6 +138,11 @@ initiate_node_upgrade()
     rm -f /etc/nova/nova.conf.d/200-crowbar-upgrade.conf
     rm -f /etc/neutron/neutron-openvswitch-agent.conf.d/200-crowbar-upgrade.conf
 
+    # mark openvswitch as enabled so it comes correctly after a node reboot
+    # this is due to the package upgrade deleting the systemd unit and adding a new unit for the
+    # service, which causes this new service unit to not be enabled.
+    systemctl enable openvswitch
```

Comment:
Could you check for openvswitch presence?

Also, is it possible that openvswitch was present before but disabled for some reason?

Reply:
yeah, what about systems with no openvswitch?

Reply:
See new patch, will only do this if it finds the openvswitch in the list of packages to update